2012-01-24	Magnus Granberg	<zorry@gentoo.org>

		* gcc/common.opt		Add -nopie

--- a/gcc/common.opt	2011-11-23 19:51:17.000000000 +0100
+++ b//gcc/common.opt	2012-01-24 16:56:24.302224357 +0100
@@ -2280,6 +2280,9 @@ Driver
 nodefaultlibs
 Driver
 
+nopie
+Driver
+
 nostartfiles
 Driver
 
